components/Header/Header.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return (
+    <StyledHeader>
+      <StyledH1>Movie-Box</StyledH1>
+    </StyledHeader>
+  );
+}
+
+const StyledHeader = styled.header`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0 1rem;
+  height: 5rem;
+  position: sticky;
+  z-index: 100;
```

Comment:
I noticed that your `position: sticky` is not working as you haven't set `top: 0` yet. 

If you want your header always fixed at the top, I would suggest to use `position: fixed` instead of `position: sticky` here. 💡 Keep in mind, that you will also then need to set a `margin-top` for your `main` element, otherwise some of your content will be hidden by your header. 

You should also erase the `z-index` as you do not need it here.  🙂 

```suggestion
  position: fixed;
  top: 0;
```